recipes/deploy-admin.rb

```diff
@@ -10,6 +10,7 @@
 shared_storage_root = get_shared_storage_root
 rest_auth_info = get_rest_auth_info
 admin_user_info = get_admin_user_info
+stack_name = stack_shortname
```

Comment:
Rute, where does the stack_shortname come from? Does it come from the cluster config's stack.name? 
ie. 
{
  "version": 102,
  [...]
   "stack": {
    "name": "kdolan-dev",